srv/web/src/components/forms/TakerAccept.tsx

```diff
@@ -39,6 +47,16 @@ export const TakerAccept = () => {
             setShowError(true)
             return
         }
+
+        const accs = new ACCS(fields.bitcoinPrivateKey)
+        const config = {} as ACCSConfig
+        //config.txType = fields. -  does this even belong to the ProposerAccept? 
+        config.privateKeyBTS = fields.bitsharesPrivateKey
+        //config.keyPairCompressedBTC = fields.bitcoinPublicAddress
+        config.accountBTS = fields.bitsharesAccountName
+        config.amount = fields.amount // still put amout field in UI 
+        config.timelockBTC = fields.timelockDuration
+        config.secret= fields.hash // we dont have to generate a secret hash again - but secret does not accept a string, only getSecret()
         console.log(fields)
     }
     return (
```

Comment:
I added all required fields in the #feature/ui-overhaul branch, they use the refactored components and look uniform.